trade-finance-manager-ui/server/middleware/validateToken/index.js

```diff
@@ -0,0 +1,19 @@
+const api = require('../../api');
+const destroySessionAndRedirectToStart = require('../../helpers/destroy-session-and-redirect-to-start');
+/**
+ * Global middleware to validate user session
+ * @param {Object} req Request object
+ * @param {Object} res Response object
+ * @param {Object} next Next object
+ */
+const validateToken = async (req, res, next) => {
+  const { userToken } = req.session;
+
+  if (await api.validateToken(userToken)) {
+    next();
+  } else {
+    destroySessionAndRedirectToStart(req, res);
+  }
+};
+
```

Comment:
Same as above

Reply:
FYI this is from `portal` microservice, maybe it has Unit tests already.